### PR TITLE
fix: [CDS-46382]: Issue with Jenkins dropdowns are handled

### PR DESCRIPTION
**Pull request by satyam-harness**
### Summary
https://harness.atlassian.net/browse/CDS-46872
https://harness.atlassian.net/browse/CDS-46382

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger jest

File: cypress/integration/70-pipeline/JenkinsStep.spec.ts [hunk 1]
```diff
@@ -95,14 +95,14 @@ describe('Connectors list', () => {
     cy.contains('span', 'Save').click()
   })
 
-  it('jenkins step addition, with jobName as runtime values', () => {
+  it.skip('jenkins step addition, with jobName as runtime values', () => {
```
Comment (kapilkumawat86) [inline cypress/integration/70-pipeline/JenkinsStep.spec.ts:98]: why are we skipping, are you giving this

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
retrigger jest

File: cypress/integration/70-pipeline/JenkinsStep.spec.ts (hunk 1 above)
Comment (satyam-harness) [reply cypress/integration/70-pipeline/JenkinsStep.spec.ts]: fixed

**Review by kapilkumawat86 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

**Pull request merged**